onboarding-manager/pkg/cloudinit/infra.cfg

```diff
@@ -199,6 +199,64 @@ datasource:
                  {{- else }}
                  Subsystem sftp /usr/lib/openssh/sftp-server
                  {{- end }}
+              - path: /etc/cron.emf/logrotate
+                permissions: '0755'
+                content: |
+                 #!/bin/sh
+                 /usr/sbin/logrotate /etc/logrotate.conf
+                 EXITVALUE=$?
+                 if [ $EXITVALUE != 0 ]; then
+                   /usr/bin/logger -t logrotate "ALERT exited abnormally with [$EXITVALUE]"
+                 fi
+                 exit $EXITVALUE
+              - path: /etc/logrotate.d/rsyslog
+                permissions: '0644'
+                content: |
+                 /var/log/syslog
+                 /var/log/mail.info
+                 /var/log/mail.warn
+                 /var/log/mail.err
+                 /var/log/mail.log
+                 /var/log/daemon.log
+                 /var/log/kern.log
+                 /var/log/auth.log
+                 /var/log/user.log
+                 /var/log/lpr.log
+                 /var/log/cron.log
+                 /var/log/debug
+                 /var/log/messages
+                 {
+                   rotate 4
+                   weekly
+                   missingok
+                   notifempty
+                   size 1G
+                   compress
+                   delaycompress
+                   sharedscripts
+                   postrotate
+                   /usr/lib/rsyslog/rsyslog-rotate || true
+                   endscript
+                 }
+              {{- if eq .FIREWALL_PROVIDER "ufw" }}
+              - path: /etc/logrotate.d/ufw
+                permissions: '0644'
+                content: |
+                 /var/log/ufw.log
+                 {
+                   rotate 4
+                   weekly
+                   missingok
+                   notifempty
+                   size 1G
+                   compress
+                   delaycompress
+                   sharedscripts
+                   postrotate
+                   [ -x /usr/lib/rsyslog/rsyslog-rotate ] && /usr/lib/rsyslog/rsyslog-rotate || true
```

Comment:
Is there a reason for the difference in the use of `/usr/lib/rsyslog/rsyslog-rotate` in this logrotate job and the one of the syslog above?